apps/src/lib/node/ledger/ethereum_oracle/mod.rs

```diff
@@ -444,7 +444,6 @@ async fn process<C: RpcClient>(
         let sig = codec.event_signature();
         let addr: Address = match codec.kind() {
             EventKind::Bridge => config.bridge_contract.into(),
-            EventKind::Governance => config.governance_contract.into(),
```

Comment:
wait, do we no longer have a separate governance contract?

Reply:
nope. we have a single `Bridge` contract